client/src/components/employer-form/index.js

```diff
@@ -52,19 +54,37 @@ export const Form = ({ initialValues, isDisabled }) => {
     // Site HCAP request
     hcswFteNumber: '',
 
-    // TODO - business detail info from mockup
-    businessKind: '',
-    workersSize: '',
-    employerType: '',
+    // Workforce Baseline
+    workforceBaseline: {},
   };
 
+  const mapBaselineList = (values) => {
+    let newWorkforceBaseline = [];
+
+    mapValues(values.workforceBaseline, (value, key) => {
+      if (value.add) {
+        newWorkforceBaseline.push({
+          role: key,
+          currentFullTime: value.currentFullTime,
+          currentPartTime: value.currentPartTime,
+          currentCasual: value.currentCasual,
+          vacancieFullTime: value.vacancieFullTime,
```

Comment:
should this be _vacancies_ or _vacancy_ maybe? Looks like it was meant to be plural and there's a typo (the issue is present in BaselineList also, could probably do a global search/replace)

Reply:
good catch!